File: UVa_11461.py
import math

while True:
    n1, n2 = input().split()
    if n1 == n2 == "0":
        break
    n1, n2 = int(n1), int(n2)
    
    count = 0
    
    for i in range(n1, n2+1):
        sqrt_ = math.sqrt(i) 
        if sqrt_ == math.floor(sqrt_):
            count += 1 
            
    print(count)

File: 114xx/UVa_11461.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    double i, n1, n2;
    
    while(1)
    {
        cin >> n1 >> n2;
        if (n1 == 0 && n2 == 0)
            break;
        
        int count = 0;
        
        for(i=n1; i <= n2; ++i)
        {
            if (sqrt(i) == floor(sqrt(i)))
                ++count;
        }
        
        cout << count << endl;

    }
    
    return 0;
}
